src/components/calendar/Modal.jsx

```diff
@@ -18,6 +18,11 @@ const Modal = ({ event, setEvent }) => {
               hour: "2-digit",
               minute: "2-digit",
             })}
+          {" - "}
```

Comment:
also, add a feature that checks if it's all day events. If it says then output "All Day"